components/search/SearchBar.vue

```diff
@@ -117,7 +117,6 @@ function onEnter() {
   // insert search term in history
   searchSuggestionRef.value?.insertNewHistory()
   emits('enter')
-  emits('redirect')
```

Comment:
Why was this removed?

Reply:
This is a duplicate event because the parent component has both @enter and @redirect handlers, and the redirect logic has already been handled in @enter, so emitting the redirect again here would result in a duplicate redirect.

Reply:
then we can remove @redirect beause it won't be firing a v-on redirect event anymore.

Reply:
Ok. code seems good 👍